Level2/Voting/README.md

```diff
@@ -35,7 +35,10 @@ All the deployed contracts-
 1) partipant1_vote - it is used to gave votes to the Participant 1
 2) partipant2_vote - it is used to gave vote to the Participant 2
 3) declare_winner - declare's the winner of the poll (only declared by the owner )
-4) parti1_cnt_vote - counts the vote of first participant (only acessible by the owner )
-5) parti2_cnt_vote - counts the vote of second participant (only acessible by the owner )
+4) partipant1 - gives the address of first participant
+5) partipant2 - give the address of second participant
+6) parti1_cnt_vote - counts the vote of first participant (only acessible by the owner )
+7) parti2_cnt_vote - counts the vote of second participant (only acessible by the owner )
+
 

```

Comment:
8)owner - This is used to check the owner address 

@AnmolSirola include this last point